parking-lot-help-request/firmware/README.md

```diff
@@ -0,0 +1,99 @@
+```json
+{"req":"env.default","name":"parking_lot_help_request_config_script_version","text":"1.1.1"}
+{"req":"hub.set","product":"com.your-company.your-name:your_product", "sn":"preferred_device_name_here"}
+{"req":"hub.set","mode":"minimum","outbound":1440,"inbound":10080}
+{"req":"card.aux","mode":"neo-monitor"}
+{"req":"card.voltage","mode":"lipo"}
+{"req":"card.motion.mode","stop":true}
+```
+6. Upload the configuration script to the Notecard via the Notecard CLI by running the following command from the command line:
+```bash 
+$ notecard -setup parking-lot-help-request-config-script.json`
+```
+7. This will execute each of the JSON commands line by line against the Notecard and then it should be all set as far as firmware configuration goes.
+8. To confirm your Notecard is now attached to your project in Notehub, navigate to the project in the browser, click the **Devices** tab and see if your Notecard has appeared. If all goes according to plan, you'll see a device with the preferred name of your Notecard (provided by the setup script) under the **Best ID** column.
+  
+![Notecard registered to Notehub project with custom ID displayed as best ID instead of device's serial number](readme-notehub-device.png)
+_Devices tab on Notehub project showing Notecard's registered to this project._
+
+If you'd prefer to upload this script using the in-browser Notecard Playground instead of downloading the CLI, simply connect your Notecarrier to your computer via the USB-A to micro USB connector, pair it via serial port in your browser by clicking the **USB Notecard** button in the [Notecard Playground](https://dev.blues.io/notecard-playground/), and copy/paste the whole config file like in the screenshot below. (Don't forget to replace the placeholder `product` and `sn` variables in the script before copy/pasting.)
+
+![Pasting in the Notecard config script in the in-browser Notecard playground](./readme-notecard-playground-script.png)
+
+Now, click the button a few times and hopefully you'll see the LED light on the board flash and the button press data similar to this screenshot below flowing into your own Notehub project.
+
+![_button.qo events registering in Notehub when the help request button is pressed on the device](readme-notehub-events.png)
+_The `_button.qo` events are the events we'll want to use to trigger SMS alerts._
+
+### Transform Notehub data with JSONata for Twilio
+
+After button press events are successfully being sent to your Notehub project, they'll need to be transformed and routed out to Twilio to trigger SMS alerts to a user's phone.
```

Comment:
Might be worth having a note somewhere in this writeup that says something like, although we’re using Twilio, you can route your button presses to any service you’d like, and set up whatever other notification system you have in mind.